### PR TITLE
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e

### PR DESCRIPTION
**Pull request by xbpiao**
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e
(win32 support  android-ndk-r9d android-ndk-r10d android-ndk-r10e android-ndk-r11c android-ndk-r13b android-ndk-r14b)

### CODE REVIEW

**Comment by xbpiao** [comment]
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e
(win32 support  android-ndk-r9d android-ndk-r10d android-ndk-r10e android-ndk-r11c android-ndk-r13b android-ndk-r14b)

macOS下android-ndk-r13b android-ndk-r14b生成时cocos2dx_physics3d.ini会出错其它正确，目前只能测试到这些，其它NDK版本未测试，但相比原来的版本兼容性已经好很多。

File: tools/tolua/cocos2dx_3d.ini [hunk 1]
```diff
@@ -7,10 +7,10 @@ prefix = cocos2dx_3d
 # all classes will be embedded in that namespace
 target_namespace = cc
 
-android_headers = -I%(androidndkdir)s/platforms/android-14/arch-arm/usr/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.7/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.7/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.8/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.8/include
+android_headers = -I%(androidndkdir)s/platforms/android-14/arch-arm/usr/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/include
```
Comment (dumganhar) [inline tools/tolua/cocos2dx_3d.ini]: There is a duplicated of `-I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/libs/armeabi-v7a/include`.
The old one is using 4.7 and 4.8, but currently, it uses `gnu_libstdc_version` variable. So we could remove one.

Comment (dumganhar) [reply tools/tolua/cocos2dx_3d.ini]: The same problem in other ini files.

Comment (xbpiao) [reply tools/tolua/cocos2dx_3d.ini]: Thanks for the reminder that you are correct and now delete the duplicate configuration.

**Comment by xbpiao** [comment]
咦！感觉这样修改兼容性好很多的。

**Comment by minggo** [comment]
Sorry, i may close it by mistook. And i can not reopen it. Could you please send another PR? Thanks.

**Comment by dumganhar** [comment]
@xbpiao, v3.15 branch has been deleted. So this pull request was automatically closed.
Could you send it to v3 branch again? Thanks.

**Comment by xbpiao** [comment]
@dumganhar @minggo 
#17817

**Pull request closed without merging**